### ISSUE TITLE
Filtering off Log-Messages in Formatter not possible

### ISSUE THREAD

**Issue by jbojbo** [TheAppleFreak/winston-slack-webhook-transport#15]
Version: 2.1.0

When i try to only return text for the defined log-level like this:

```
this.logger.add(
        new SlackHook({
          webhookUrl: slackHookURl,
          level: 'verbose',
          formatter: (info) => {
            if (info.level == 'verbose') {
              return { text: info.message };
            }
          },
        }),
      );
```

the Hook will not work anymore.
Only if i return something in every other case, it starts working again.

```
this.logger.add(
        new SlackHook({
          webhookUrl: slackHookURl,
          level: 'verbose',
          formatter: (info) => {
            if (info.level == 'verbose') {
              return { text: info.message };
            }
            return { text: ' ' };
          },
        }),
      );
```

**Comment by TheAppleFreak** [comment]
Crap, I did not realize this issue existed... apologies for the delay!

That's an interesting issue there... I suppose if we have a default payload in the event nothing is returned that should address it? I'll play around with it and see what can be done.

**Comment by TheAppleFreak** [comment]
Fixed in v2.2.3. Thanks again to @ArturAralin for the fix!
